### DISCUSSION TITLE
Community

### DISCUSSION THREAD

**Discussion by mrguamos** [jooby-project/jooby#2996] [Q&A] [answered]
Do we have a discord channel or something like that?

**Comment by SentryMan** [comment]
Allegedly there is a slack, but I'm totally unable to access it

**Reply by mrguamos** [reply]
would be nice to have one

**Reply by rjaros** [reply]
I have problems accessing slack channel as well. An existing account seems to be required.

**Reply by edgar-espina-wpp** [reply]
what is the issue with slack?

**Reply by rjaros** [reply]
I enter my email address, click "Sign in with email", the page ask for the code, but I receive an email with the following message:
```
Is this the right email?
It looks like there isn’t an account on jooby tied to this email address.
```

**Reply by rjaros** [reply]
"Sign in with google" doesn't work as well.

**Reply by SentryMan** [reply]
yeah that's my issue as well

**Reply by edgar-espina-wpp** [reply]
That's odd... should I create a discord channel? I think there was one

**Reply by jknack** [reply]
Here is: https://discord.com/channels/1225457509909922015

**Reply by agentgt** [reply]
@jknack @edgarespinawt I clicked on the link and it says I don't have access.

**Reply by jknack** [reply, marked as answer]
@agentgt try this one: https://discord.gg/nmfJmmrq

**Reply by zoltan-kollar** [reply]
The above Discord invite seems to be expired. Is the channel still open for public?

<img width="469" height="357" alt="Screenshot 2025-10-17 at 23 57 16" src="https://github.com/user-attachments/assets/16157375-ba7a-4a16-b340-7469a1026445" />

**Reply by jknack** [reply]
Try this one: https://discord.gg/JmyxrKPvjY, is the one from README.md

**Reply by zoltan-kollar** [reply]
That worked, thank you. I have opened https://github.com/jooby-project/.github/pull/1 and https://github.com/jooby-project/jooby/pull/3789 to adopt this link in all places.
